stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -69,7 +143,185 @@ private static String getIdAttribute(View element) {
 
   @Override
   public View getViewForHighlighting(Object element) {
-    return (View)element;
+    return (View) element;
+  }
+
+  @Override
+  protected void onGetStyles(View element, StyleAccumulator styles) {
+
+    for (ViewCSSProperty property : getProperties()) {
+      Object value = null;
```

Comment:
recommend not initializing this to `null` here. instead, set it to null in the error paths (within the `catch` clauses). That makes is easier to determine that every code path is assigning to this and that it didn't get forgotten.